### PR TITLE
Fix python 3.5 build.

### PR DESCRIPTION
**Pull request by ueshin**
Currently Python 3.5 build in travis-ci is not working properly since the Python is upgraded while setting up the environment because the dev dependency is strictly versioned but some of them depends only Python 3.6 or upper.

E.g., this (https://travis-ci.com/databricks/koalas/jobs/197751790) is the 3.5 build, but we can see:

Screen Shot 2019-05-06 at 20 05 04

### CODE REVIEW

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -1346,7 +1346,7 @@ def sort_values(self, by, ascending=True, inplace=False, na_position='last'):
               for colname, asc in zip(by, ascending)]
         kdf = DataFrame(self._sdf.sort(*by), self._metadata.copy())
         if inplace:
-            self._sdf: spark.DataFrame = kdf._sdf
+            self._sdf = kdf._sdf
```
Comment (ueshin) [inline databricks/koalas/frame.py]: Variable Annotations is supported from Python 3.6. https://www.python.org/dev/peps/pep-0526/

**Comment by arisehkawamura** [comment]
When will this fix be released to PyPI?

**Comment by HyukjinKwon** [comment]
0.3.0 is released now.

**Comment by arisehkawamura** [comment]
Thank you!

**Comment by thunterdb** [comment]
@ueshin we lost a few annotations in the process. Can you add them as comments (`... # type: ks.DataFrame`) so that mypi can still pick up the types? Thanks!

**Pull request merged**